db/migrate/20250508174727_add_cascade_delete_to_event_procedures.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddCascadeDeleteToEventProcedures < ActiveRecord::Migration[7.1]
```

Comment:
I am not used to this approach, usually we do this at model level using dependent destroy on the relationship.
We need to make sure, no data related to the user remains there.

Is there any specific reason for doing this at db level? I believe this way we skip model hooks if there are any. I think we don;t have, but anyway it is good to avoid it.